src/Tizen.Data.Tdbc/Tizen.Data.Tdbc/Sql.cs

```diff
@@ -0,0 +1,124 @@
+﻿/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.Data.Tdbc
+{
+    /// <summary>
+    /// Represents a SQL query string.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public class Sql
+    {
+        private string _sql;
+        private Dictionary<string, Tuple<Object, Type>> _bindings = new Dictionary<string, Tuple<object, Type>>();
+
+        /// <summary>
+        /// The SQL command string.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        public string Command
+        {
+            get => _sql;
+        }
+
+        /// <summary>
+        /// The bindings as a dictionary of command.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        public Dictionary<string, Tuple<Object, Type>> Bindings
+        {
+            get => _bindings;
```

Comment:
I wanted to suggest cleaner interface, below. But an Sql driver uses that, so the data should be available to external modules.
```suggestion
        /// The bindings as a name to value Dictionary.
        /// </summary>
        /// <since_tizen> 11 </since_tizen>
        public IReadOnlyDictionary<string, Object> Bindings
        {
            get => _bindings.ToDictionary(item => item.Key, item => item.Value.Item1);
```
Then I guess it will have to stay.

Reply:
I changed dictionary as `IReadOnlyDictionary<string, Object> Bindings`. The former type `Dictionary<string, Tuple<Object, Type>> Bindings`  is for storing data type of bindings, but i found that the data type can be obtained by using `Object.GetType()` method. Please refer `Tizen.Data.Tdbc.Driver.Sqlite.Statement.Prepare()`.